### DISCUSSION TITLE
Unable to press F8 key on iOS

### DISCUSSION THREAD

**Discussion by woodfairy** [utmapp/UTM#6604] [UTM Help (iOS)]
I am unable to press F8 key on the iOS client. There are no F-Keys on the keyboard. 

**Configuration (required)**
* UTM Version: latest (from AltStore PAL)
* OS Version: 17.5.1 
* Device Model: iPhone 15 Pro
* Is it jailbroken (name jailbreak used)? No
* How did you install UTM? AltStore PAL

**Comment by osy** [comment]
Open the on screen keyboard. In the bar above the keyboard there are extra keys. You can scroll right to find F8
